main/test/ca/uwaterloo/flix/library/TestMpmc.flix

```diff
@@ -16,65 +16,65 @@
  */
 
 @test
-def testNew01(): Channel.Mpmc & Impure = Channel.new(0)
```

Comment:
But I think these should belong in `Channel`...

Reply:
The tests? or new?